src/ensembl/src/content/app/browser/browserReducer.ts

```diff
@@ -80,7 +81,11 @@ export function browserEntity(
       const genomeIdToRemove = action.payload;
       const activeGenomeId = state.activeGenomeId;
       if (activeGenomeId === genomeIdToRemove) {
-        return { ...state, activeGenomeId: null };
+        return {
+          ...state,
+          activeGenomeId: null,
+          activeEnsObjectIds: omit(state.activeEnsObjectIds, activeGenomeId)
```

Comment:
I would advise against using `omit`. It sure looks nice, but lodash is planning to remove it from its next major release (whenever _that_ happens). [Link to the roadmap](https://github.com/lodash/lodash/wiki/Roadmap#v500-2021) and [link to a discussion on github](https://github.com/lodash/lodash/issues/2930#issuecomment-272298477).

You can do pickBy; or you can do something like:

```
const updatedEnsObjectIds = { ...state.activeEnsObjectIds }
delete updatedEnsObjectIds[activeGenomeId]

/* other stuff */
return {
          ...state,
          activeGenomeId: null,
          activeEnsObjectIds: updatedEnsObjectIds
}
```

Reply:
Created a custom `omit` function as discussed.